dipy/segment/tests/test_clustering.py

```diff
@@ -388,11 +412,13 @@ def test_cluster_map_getitem():
         assert_equal(cluster_map[i], clusters[i])
 
     # Test advanced indexing
-    assert_array_equal(cluster_map[advanced_indices], [clusters[i] for i in advanced_indices])
+    assert_array_equal(
+        cluster_map[advanced_indices], [
+            clusters[i] for i in advanced_indices])
```

Comment:
This is too many lines and broken up in a way that's hard to read. How about: 

assert_array_equal(cluster_map[advanced_indices], 
                                [clusters[i] for i in advanced_indices])